### PR TITLE
[A-4088] Fixing messages bug

### PR DESCRIPTION
**Pull request by ymao1**
Issue was the server code was never updated to include messagesViewed on user object. I fixed that and made the dismiss X on the toaster darker so you can see it better.

### CODE REVIEW

File: src/server/app/admin/models/user.server.model.js [hunk 1]
```diff
@@ -153,6 +153,11 @@ let UserSchema = new GetterSchema({
 		default: Date.now,
 		get: util.dateParse
 	},
+	messagesViewed: {
```
Comment (rbutterfield) [inline src/server/app/admin/models/user.server.model.js:156]: should we also add this field to the audit copy? a user update gets audited when the messagesViewed timestamp is updated, but in the audit entry it looks like there are no changes

**Comment by rbutterfield** [comment]
when i dismiss a message i'm seeing a user update get audited and i'm not seeing the message pop up again when i refresh the UI. this behavior is fine with me, but based on our conversation i'm not sure this was the intention?

**Comment by ymao1** [comment]
@rbutterfield, when you dismissed it, was it a single message or multiple messages (i.e., 3 unread messages)? When I tested it, it looked like if a single new message popped up, dismissing it marks it as read, but if multiple messages are unviewed, dismissing it did not update the messagesViewed time, you have to click through to the welcome page. I actually thought this behavior made sense, so I left it as is.

File: src/server/app/admin/models/user.server.model.js (hunk 1 above)
Comment (ymao1) [reply src/server/app/admin/models/user.server.model.js:156]: done

**Comment by rbutterfield** [comment]
@ymao1 that's fine with me

**Review by rbutterfield — APPROVED** [review]

**Review by bodnarbm — APPROVED** [review]

**Pull request merged**